js/customize-concurrency.js

```diff
@@ -69,10 +69,12 @@
 								control.container.on( 'click', '.concurrency-conflict-override', function() {
 									control.setting.concurrency_override = true;
 									control.notifications.remove( control.setting.id + ':concurrency_conflict' );
+									alert( control.setting.id + ':concurrency_conflict' );
 								} );
 								control.container.on( 'click', '.concurrency-conflict-accept', function() {
 									control.setting.set( theirValue );
 									control.notifications.remove( control.setting.id + ':concurrency_conflict' );
+									alert( control.setting.id + ':concurrency_conflict' );
```

Comment:
Added this because the buttons stopped working but I think that is because `[customize-object-selector-control] The post_query_args arg is deprecated in favor of post_query_vars.` So hopefully an update to the BOD repo will take care of that. Ignoring for now.